pkg/ctl/sender/client_docker.go

```diff
@@ -37,13 +36,13 @@ func NewContainerisedClient(apiVersion string, containerID string) (c Client, er
 
 	cl, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
 	if err != nil {
+		clog.WithError(err).Error("Failed to initialize docker client.")
 		err = errors.Wrap(err, "failed to initialize docker client")
 		return
```

Comment:
do we really want to log and return the error as well?

Reply:
Will remove that, just wanted to log the error here for the unit test.